app/controllers/api/v2/interfaces_controller.rb

```diff
@@ -93,6 +94,24 @@ def destroy
         process_response @interface.destroy
       end
 
+      def parse_search_string
```

Comment:
Why not add a `scoped_search` definition on the base nic model and using `resource_scope_for_index` in the index method, as we do for other controllers, instead of manually parsing the search string?
That way plugins can also add their definitions, for example for remote execution interface, and more complex searches can be parsed as well.

Reply:
yes @tbrisker , will update with the same.